recipes/libgeotiff/all/conanfile.py

```diff
@@ -43,7 +43,8 @@ def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("libtiff/4.4.0")
+        # libgeotiff/include/xtiffio.h includes libtiff/include/tiffio.h
+        self.requires("libtiff/4.5.1", transitive_headers=True, transitive_libs=True)
```

Comment:
`transitive_libs` is a bit more rare to need. Can you give some insight as to why it's needed here? Thanks :)

Reply:
Since the public header of libgeotiff exposes symbols from libtiff without the integrator ever including a public header of libtiff, I think transitive_libs should be True.
The test package is in this case: it does not include a public header of libtiff directly and fails to build locally if I set this to False.

Reply:
Thanks! Explanations like this help specially for future maintenance, thanks for the extra insight :)